### ISSUE TITLE
Val metrics (a1, a2, a3) almost 0 and don't improve despite loss converging - DepthFormer

### ISSUE THREAD

**Issue by alimtaha** [zhyever/Monocular-Depth-Estimation-Toolbox#32]
DepthFormer doesn't appear to be training when looking at validation metrics despite the training loss converging - data is being loaded correctly, verified by visualising the depth maps before training loop in IterBasedRunner. Implemented warm_up=True in decode head decode loss but still seeing no improvement.

**Comment by zhyever** [comment]
Yes, I'm confused now. The convergence problems seem not to be solved by a simple warm_up strategy. I observe this issue in all the methods I implement. Moreover, even the official repos of bts/adabins/transdepth suffer a similar problem. Specifically, I think the gradience is vanishing.

Now, a direct solution is to run the script several times. If the loss is converging in the first several steps, the following training stage will be ok. I would appreciate it if anyone can give me a hand to solve this issue.

**Comment by zhyever** [comment]
Here is one possible solution: modify the schedule to make the training stage start with a relatively small learning rate.

**Comment by zhyever** [comment]
Close for now.
